nemo_skills/dataset/aime25-de-prompt-de/prepare.py

```
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Ruff (0.14.14)

The file is executable but no shebang is present

(EXE002)